README.rst

```diff
@@ -137,9 +172,10 @@ information.
    :target: https://codecov.io/github/coala-analyzer/coala?branch=master
 .. |Documentation Status| image:: https://readthedocs.org/projects/coala/badge/?version=latest
    :target: http://coala.rtfd.org/
-.. |https://gitter.im/coala-analyzer/coala| image:: https://img.shields.io/badge/gitter-join%20chat%20%E2%86%92-brightgreen.svg
-   :target: https://gitter.im/coala-analyzer/coala
 .. |AGPL| image:: https://img.shields.io/github/license/coala-analyzer/coala.svg
    :target: https://www.gnu.org/licenses/agpl-3.0.html
 .. |Gitmate| image:: https://img.shields.io/badge/Gitmate-0%20issues-brightgreen.svg
    :target: http://gitmate.com/
+.. |gitter| image:: https://badges.gitter.im/coala-analyzer/coala.svg
```

Comment:
why does this gitter thingie appear on both Support and Getting Involved sections? looks kinda ugly. should keep only one of them maybe :D ?